modules/programs/hwatch.nix

```
@@ -0,0 +1,50 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkIf
+    mkOption
+    types
+    ;
+  cfg = config.programs.hwatch;
+in
+{
+  meta.maintainers = with lib.hm.maintainers; [
+    Aehmlo
+  ];
+
+  options.programs.hwatch = {
+    enable = lib.mkEnableOption ''
+      hwatch, a modern alternative to the {command}`watch` command
+    '';
+
+    package = lib.mkPackageOption pkgs "hwatch" { nullable = true; };
+
+    extraOptions = mkOption {
```

```suggestion
    extraArgs  = mkOption {
```
Or something like this?

---

Changed to `extraArgs` for consistency.